KrampitzKreutzwiserSettlersOfCatan/src/krampitzkreutzwisersettlersofcatan/GameFrame.java

```diff
@@ -25,7 +31,14 @@ public GameFrame(MainMenu m) {
      */
     private void initFrame() {
         setTitle("Settlers of Catan");
-        setSize(1920, 1080); //set the size to 1080p
+        setSize(1920, 1080); //set the size to 1080p as a back up      
+        //setSize(1080, 1920);
+        //setSize(1280, 720);

```

Comment:
These are just testing resolutions I found them useful to make sure everything works okay